app/views/research_sessions/questions/methodologies.html.erb

```diff
@@ -3,76 +3,80 @@
-      choices_selected: @research_session.methodologies,
-      legend: 'How will you be gathering information?',
-      choice_type: 'multi',
-      errored: @research_session.errors['methodologies'].present?,
-      validation: @research_session.errors['methodologies'][0]
-    ) %>
-
-    <%= c('ChoiceControlledContent',
-      is_active: (@research_session.methodologies.present? && @research_session.methodologies.include?('other')),
-      controlled_by: '[id="research_session[other]"]') do %>
-      <%= c('SinglelineTextControl',
-        id: 'research_session[other_methodology]',
-        name: 'other_methodology',
-        value: @research_session.other_methodology,
-        label: t('research_session_attr_labels.other_methodology'),
-        size: 48,
-        errored: @research_session.errors['other_methodology'].present?,
-        validation: @research_session.errors['other_methodology'][0]
+  <div class="Step-form">
+    <%= c('Form', action: wizard_path, patch: true) do %>
+      <%= c('OfManyChoiceControl',
+        id: 'research_session[methodologies][]',
+        name:'methodologies',
+        collection: "research_session[#{:methodologies}][]",
+        choices: Methodologies::CHOICES,
+        choices_selected: @research_session.methodologies,
+        legend: 'How will you be gathering information?',
+        choice_type: 'multi',
+        errored: @research_session.errors['methodologies'].present?,
+        validation: @research_session.errors['methodologies'][0]
       ) %>
-    <% end %>
 
-    <%= render partial: 'submit_step' %>
-  <% end %>
-
-  <%= c('PreviewConsentForm') do %>
-    <%= c('HeadingOne', text: t('preview.able_to_consent.what_happens_in_this_research_session.heading')) %>
+      <%= c('ChoiceControlledContent',
+        is_active: (@research_session.methodologies.present? && @research_session.methodologies.include?('other')),
+        controlled_by: '[id="research_session[other]"]') do %>
+        <%= c('SinglelineTextControl',
+          id: 'research_session[other_methodology]',
+          name: 'other_methodology',
+          value: @research_session.other_methodology,
+          label: t('research_session_attr_labels.other_methodology'),
+          size: 48,
+          errored: @research_session.errors['other_methodology'].present?,
+          validation: @research_session.errors['other_methodology'][0]
+        ) %>
+      <% end %>
 
-    <%= c('Paragraph') do %>
-      <%= raw t('preview.able_to_consent.what_happens_in_this_research_session.paragraphs')[0] % [
-        c(
-          'Output',
-          text: @research_session.researcher_name,
-          field: 'researcher_name',
-          url: edit_link_for(:researcher_name)
-        ),
-        c(
-          'Output',
-          text: @research_session.researcher_job_title,
-          field: 'researcher_job_title',
-          url: edit_link_for(:researcher_job_title)
-        )
-      ] %>
+      <%= render partial: 'submit_step' %>
```

Comment:
This can be written as <%= render 'submit_step' %>

Reply:
Could this be part of the Ruby refactor?

Reply:
Certainly could be 👍